beta/kustomize/README.md

```diff
@@ -0,0 +1,214 @@
+
+- **Actor**: The email address of the administrator in Google Workspace that the service account is acting on behalf of.
+- **Bridge Address**: The URL you will use for your SCIM bridge (not your 1Password account sign-in address). This is most often a subdomain of your choosing on a domain you own. If you're using the Let's Encrypt overlay, this would be the value of `OP_TLS_DOMAIN` configured in `patch-configmap.yaml`. It's possible `OP_TLS_DOMAIN` has not been configured yet. If so, use the intended value of `OP_TLS_DOMAIN` as the value for the Bridge Address.
+
+## Step 4: Prepare the deployment
+
+To configure the deployment for use, edit the file
+`./overlays/deployment/kustomization.yaml` and make changes as appropriate. For
+example: if you're going to be using a self-managed TLS certificate, then comment out
+the `letsencrypt` overlay and instead uncomment the `self-managed-tls` overlay.
+
+First, run the following commands to create a namespace for the SCIM bridge, as well
+as switch to that namespace:
+
+```bash
+kubectl create namespace op-scim
+kubectl config set-context --current --namespace=op-scim
+```
+
+Next, run the following command while in the Root Folder to deploy your bridge:
+
+```bash
+kubectl apply -k ./overlays/deployment
+```
+
+## Step 5: Final configuration
+
+If using Let's Encrypt, it's necessary to create the deployment _first_, and then
+configure the domain that 1Password SCIM Bridge should request a certificate for.
+
+After your initial deploy, run `kubectl get service`. The output will show three services,
+one with the type of LoadBalancer. If an external IP address is pending for the
+LoadBalancer, run the command again until an external IP address is shown. This IP
+address should be configured as the A record of your DNS record, as mentioned in 
+the Preparation document. As an example, it could be `scim.example.com`, where
+`example.com` is a domain you control.
+
+After creating the DNS record with your DNS provider, edit the file 
+`patch-configmap.yaml` in the Let's Encrypt overlay folder. Replace the value
+`scim.example.com` with the A record created. Save this file.
+
+From the Root Folder, redeploy the bridge using the command listed in step 4. The
+SCIM bridge will restart using this new deployment, and configure the Let's Encrypt
+integration to issue and use a certificate.
+
+## Step 6: Connect your identity provider
+
+To finish setting up automated user provisioning, [connect your identity provider to the SCIM bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider).
+
+## Update your SCIM bridge
+
+👍 Check for 1Password SCIM Bridge updates on the [SCIM bridge release page](https://app-updates.agilebits.com/product_history/SCIM).
+
+To update the SCIM bridge, connect to your Kubernetes cluster and run the following command, replacing v2.8.0 with the latest version:
+
+```bash
+kubectl set image deploy/op-scim-bridge op-scim-bridge=1password/scim:v2.8.3
```

Comment:
Since we are not having the deployment go to the default namespace, we should probably target the namespace we recommended the customer create.

```suggestion
kubectl set image deploy/op-scim-bridge op-scim-bridge=1password/scim:v2.8.3 -n=op-scim
```

Reply:
Woops, good catch, thanks.

Reply:
Good catch, thanks.